src/components/Utils/Utils.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { StoryCard } from '../.storybook';
+import './utils-stories.scss';
+
+export default {
+  title: 'Utils/Default',
+  parameters: {
+    previewTabs: {
+      'storybook/docs/panel': { hidden: true }
+    },
+    options: { showPanel: false }
+  }
+};
+
+const HOOKS = ['useDidMount', 'useDidUpdate', 'usePrevious', 'usePropsState', 'useWillUnmount'];
+
+const FORMAT_UTILS = ['formatWei', 'formatBigNumberString', 'formatFiat', 'formatPercentage', 'formatTimestamp'];
+
+const UTILS = [
+  {
+    title: 'Hooks',
+    utils: HOOKS
+  },
+  {
+    title: 'Format utils',
+    utils: FORMAT_UTILS
+  }
+];
+
+export const Default = () => {
+  return UTILS.map(({ title, utils }) => (
+    <StoryCard key={title} title={title} className="utils-stories__card">
+      <ul>
+        {utils.map(hook => (
+          <li key={hook} className="utils-stories__list-item">
+            {hook}
+          </li>
+        ))}
+      </ul>
+    </StoryCard>
+  ));
+};
```

Comment:
I don't think this is the best way for us to show these functions, as it doesn't provide much more information than "these functions exist". Non UI elements shouldn't have UI stories - we should [use MDX](https://stackoverflow.com/questions/70365222/is-it-possible-to-write-doc-for-function-class-non-ui-elements-in-storybook) instead.

What would be ideal, is if we split them up into sections. For example, we should have a `Formatting` section under `Utils`, which has the `format...` functions.

For each section, we should list each function and what its output looks like.